adapters/primitive-adapters/build.gradle

```diff
@@ -6,8 +6,6 @@ plugins {
   alias(libs.plugins.binaryCompatibilityValidator)
 }
 
-archivesBaseName = 'sqldelight-primitive-adapters'
```

Comment:
What's the reason for this line? It does not change the name of the jar file: https://repo.maven.apache.org/maven2/app/cash/sqldelight/async-extensions/2.0.2/async-extensions-2.0.2.jar

Reply:
It dictates the JS module name

Reply:
The reason I removed this line at all was a problem with the BCV plugin that was unable to merge the platform specific api files into one klib api file due a mismatch in the header, line 8, so I removed the `archivesBaseName`. But if this name is required, I will create an issue.

Reply:
There's a dedicated property for setting the module name now which we can use instead. Example: https://github.com/cashapp/redwood/blob/11b306929c0c9989f89ba96735aa8bb57ddebf0a/test-app/browser/build.gradle#L10

Reply:
Hm, now I get the next error, `moduleName` is not a Gradle Property and it must be specified before calling `browser` or `nodeJS`, but this does not work with convention plugins.

Reply:
Just for the reference:
https://github.com/Kotlin/binary-compatibility-validator/issues/249
https://youtrack.jetbrains.com/issue/KT-71362/KGP-JS-moduleName-is-not-compatible-with-convention-plugins